### DISCUSSION TITLE
pending transaction

### DISCUSSION THREAD

**Discussion by mojtabatouzandeh** [blockscout/blockscout#9760] [General]
I made some Dijon Bridge almost an hour ago, it was deducted from my account, but the Dijon was not deposited for me. please follow up

**Comment by Markfluit** [comment]
I had similar issue few days ago while waiting and didn’t get it resolved I contacted cryptorecoverys@outlook.com 
They helped me out

**Comment by Neo1pak** [comment]
Transaction hash

0xc3687b7496d03e3e9415857e47a8de870c86aa717fb74802e872f239f32b83cd

45 mint ago and I still waiting but my 0.017 bnb not receive in my BNB chain . Please solve this issue. Thank you

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
